js/widgets/timbre.js

```diff
@@ -125,18 +125,16 @@ function TimbreWidget() {
         cell.style.maxHeight = cell.style.height;
         cell.style.backgroundColor = platformColor.selectorBackground;
 
-        cell.onmouseover = function() {
-            this.style.backgroundColor = platformColor.selectorBackgroundHOVER;
-        };
+        cell.onmouseover = () => this.style.backgroundColor = platformColor.selectorBackgroundHOVER;
```

Comment:
`style` should be the property of the `cell`. When you use arrow functions `this` refers to the class object. "ERROR"